src/dashboard/Data/Browser/PointerKeyDialog.react.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2016-present, Parse, LLC
+ * All rights reserved.
+ *
+ * This source code is licensed under the license found in the LICENSE file in
+ * the root directory of this source tree.
+ */
+import Button             from 'components/Button/Button.react';
+import Dropdown           from 'components/Dropdown/Dropdown.react';
+import Field              from 'components/Field/Field.react';
+import { footer }         from 'components/Modal/Modal.scss';
+import Label              from 'components/Label/Label.react';
+import Modal              from 'components/Modal/Modal.react';
+import Option             from 'components/Dropdown/Option.react';
+import React              from 'react';
+
+export default class PointerKeyDialod extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      name: null
+    };
+  }
+
+  async componentDidMount() {
+    const defaultPointerKey = await localStorage.getItem(this.props.className);
+    this.setState({ name: defaultPointerKey });
+  }
+
+  render() {
+    let content = null;
+    let hasColumns = this.props.currentColumns.length > 0;
+    if (hasColumns) {
+      content = (
+        <Field
+          label={
+            <Label
+              text='PointerKey' />
+            }
+          input={
+            <Dropdown
+              placeHolder='Select a column'
+              value={this.state.name}
+              onChange={(name) => this.setState({ name: name })}>
+              {this.props.currentColumns.map((t) => <Option key={t} value={t}>{t}</Option>)}
+            </Dropdown>
+          } />
+      )
+    }
+    return (
+      <Modal
+        type={Modal.Types.INFO}
+        icon='warn-outline'
+        title={'Change pointer value for class '+this.props.className}
+        subtitle={hasColumns ? 'The column will be used throughout dashboard as pointer value for this class.' : 'There are no columns on this class that can be set a pointer value.'}
+        confirmText='Update pointer value'
```

Comment:
Rename `pointer value` to `pointer key`. We should use the same term throughout code and UI.

Reply:
This is marked as resolved, but the change has not been made.